db/options.go

```diff
@@ -115,10 +76,10 @@ func WithNewEventCodec(ec core.EventCodec) NewOption {
 	}
 }
 
-// WithNewLowMem specifies whether or not to use low memory settings.
-func WithNewLowMem(low bool) NewOption {
+// WithNewToken provides authorization for interacting with a db.
+func WithNewToken(t thread.Token) NewOption {
```

Comment:
Maybe _New_ can be avoided and was biased by the deleted option? i.e: `WithToken`?

Reply:
`WithNewToken` is used with the `NewOptions` when creating a new database. `WithToken` is used with normal `Options`... open to naming suggestions there but that was the best I came up with.